deployment/helm/topology-aware/values.yaml

```diff
@@ -20,7 +20,7 @@ resources:
   memory: 512Mi
 
 nri:
-  patchContainerdConfig: false
+  patchRuntime: false
```

Comment:
Should `patchRuntime` be `patchRuntimeConfig`?

Reply:
good catch.